arctic/store/_ndarray_store.py

```diff
@@ -151,26 +152,20 @@ def _index_range(self, version, symbol, from_version=None, **kwargs):
         return from_index, None
 
     def get_info(self, arctic_lib, version, symbol, **kwargs):
+        ret = {} 
         collection = arctic_lib.get_top_level_collection()
-        dtype = self._dtype(version['dtype'], version.get('dtype_metadata', {}))
-        length = int(version['up_to'])
+        ret['dtype'] = self._dtype(version['dtype'], version.get('dtype_metadata', {}))
+        ret['length'] = int(version['up_to'])
 
         spec = {'symbol': symbol,
                 'parent': version.get('base_version_id', version['_id']),
-                'segment': {'$lt': length}}
-
-        n_segments = collection.find(spec).count()
-
-        est_size = dtype.itemsize * length
-        return """Handler: %s
-
-dtype: %s
+                'segment': {'$lt': ret['length']}}
 
-%d rows in %d segments
-Data size: %s bytes
+        ret['n_segments'] = collection.find(spec).count()
```

Comment:
This DB query may add a bit of overhead on the query perf.  We've got the `segment_count` field on version document - can probably just do that instead?